### ISSUE TITLE
cosmic-swingset "make scenario2-setup" task fails

### ISSUE THREAD

**Issue by kumavis** [Agoric/agoric-sdk#1257]
**Describe the bug**
following instructions from here https://github.com/Agoric/agoric-sdk#run-the-larger-demo
resulted in error
```
# github.com/Agoric/agoric-sdk/packages/cosmic-swingset/app
app/version_check.go:5:1: syntax error: non-declaration statement outside function body
note: module requires Go 1.14
make: *** [Makefile:151: compile-go] Error 2
```

**To Reproduce**
```bash
git clone git@github.com:Agoric/agoric-sdk.git
cd agoric-sdk
yarn && yarn build
cd packages/cosmic-swingset
make scenario2-setup
```

**Expected behavior**
`make` task completes without error

**Platform Environment**

Linux 5.0.0-38-generic #41-Ubuntu x86_64 GNU/Linux
master b2bb165c28d69fdeb18bf409e79e0598307c6b05
`@agoric/acorn-eventual-send@2.0.6-27-gb2bb165c`
node 12.16.1

**Comment by kumavis** [comment]
following instructions from here https://github.com/Agoric/agoric-sdk#run-the-larger-demo

**Comment by michaelfig** [comment]
The version_check.go is working as designed: you need Golang 1.14 to run scenario2.  However, the documentation in the top-level README.md is out of date: you should be using the Agoric CLI to start a demo Dapp, not wandering around in cosmic-swingset.

It would be good to update the README.md; it fell behind because our priority has been making the https://agoric.com/documentation/ site accurate.

A change like this to the "Run the larger demo" section would be helpful (and then Golang is not required):

## Run a basic demo

Visit https://agoric.com/documentation/ for getting started instructions.

TL;DR:

* `yarn link-cli ~/bin/agoric`
* `cd ~`
* `agoric init foo`
* `cd foo`
* `agoric install`
* `agoric start`

Then browse to http://localhost:8000
